<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Black Hole</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="pictureLayout"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_startAnimation">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Angle:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="m_angle">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>359.999900000000025</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Angle step:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="m_angleStep">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-359.999900000000025</double>
           </property>
           <property name="maximum">
            <double>359.999900000000025</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.300000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number lines:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="m_numLines">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Line lenght:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="m_lineLenght">
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Timer interval:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="m_timerInterval">
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="m_widgetWidth">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1920</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>700</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="m_widgetHeight">
           <property name="maximum">
            <number>1080</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>700</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="m_frame">
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
